compiler/plugins/target/LLVMCPU/LLVMTargetOptions.cpp

```diff
@@ -139,6 +139,7 @@ void LLVMTarget::storeToConfigAttrs(MLIRContext *context,
   if (vectorWidthInBytes != DEFAULT_VECTOR_WIDTH_IN_BYTES) {
     addInt64("native_vector_size", vectorWidthInBytes);
   }
+  addInt64("max_stack_allocation_size", maxStackAllocSizeInBytes);
```

Comment:
I would rather add it as an int64_t in target than unsigned to be consistent.

Reply:
Sure, applied